modules/nw-udn-benefits.adoc

```diff
@@ -29,4 +29,9 @@ User-defined networks provide the following benefits:
 +
 * **Network parity**: With user-defined networking, the migration of applications from OpenStack to {product-title} is simplified by providing similar network isolation and configuration options.
 
-Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is: create a namespace, create and configure the custom resource, create pods in the namespace.
\ No newline at end of file
+Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is as follows:
```

Comment:
A pod created in the UDN namespace can be configured to have a default, primary, or secondary network. What does statement "This provides the flexibility of a secondary network to the primary network" (line 16) mean?

Reply:
@weliang1 I didn't write this content, but did contact the original author who said he thinks it comes from this blurb: 

the OVN-Kubernetes CNI plugin with its rich set of features can be used for both primary and secondary networks, to mitigate that problem (additional cni plugins), but more fundamentally, the issue with secondary interfaces is that they are not "1st-class citizens" with Kubernetes networking, and limitations still exist (for example, they have limited integration with Kubernetes services). 

If we want it revised, I can do it in a separate PR.

Reply:
> @weliang1 I didn't write this content. I did contact the original author, but he also wasn't sure exactly what we meant by this. We have a PR (https://github.com/openshift/openshift-docs/pull/87260/files) that expands on L2 and L3 information. I can get this sentence removed in a separate PR because it doesn't make sense.

@stevsmit Agree to remove line 16.